adapters/rainbowkit-connector/src/index.ts

```diff
@@ -150,7 +21,7 @@ export const bloctoWallet = ({ chains }: BloctoWalletOptions): Wallet => ({
   },
   installed: true,
   createConnector: (): any => {
-    const connector = new BloctoConnector({ chains });
+    const connector = new BloctoConnector({ chains, appId });
```

Comment:
const connector = new BloctoConnector({ chains, options: { appId } });
```suggestion
    const connector = new BloctoConnector({ chains, options: { appId } });
```

Reply:
commit [8b51e19](https://github.com/blocto/blocto-sdk/pull/218/commits/8b51e19a12bb825aaff167e594f80abc70c95e2a)